
# Learning the Molecular Code of RNA Regulation

RNA regulation plays a major role in cellular function, and alterations such as alternative splicing are frequently implicated in human disease including cancer. Key players are RBPs that recognise cis-regulatory RNA sequence elements, such as short sequence motifs or structured RNA folds. However, the position and function of most cis-regulatory elements remain unknown, hindering the interpretation of disease-associated mutations and the resulting splicing changes.

We recently established a high-throughput screening approach to decode the cis-regulatory landscape of alternative splicing decisions in collaboration with Dr. Julian König and Dr. Stefan Legewie at IMB in Mainz. As a prototype example, we dissected the cancer-relevant alternative exon 11 in the proto-oncogene MST1R (RON) encoding for a receptor tyrosine kinase ([Braun et al., 2018](https://www.nature.com/articles/s41467-018-05748-7). Skipping of RON exon 11 results in the pathological isoform RON∆165 that promotes tumour invasiveness and is frequently upregulated in tumours. Mathematical modeling of splicing kinetics in the Legewie group enabled us to identify more than 1,000 mutations affecting RON exon 11 skipping. Importantly, the measured effects correlated with RON alternative splicing in cancer patients bearing the same mutations. Moreover, we highlighted the RNA-binding protein heterogeneous nuclear ribonucleoprotein H (HNRNPH) as a key regulator of RON splicing in healthy tissues and cancer. Our results offer insights into splicing regulation and the hidden layer of splicing-effective mutations in cancer.

![Figure 2: A high-throughput screen to map splice-regulatory mutations for alternative exon 11 in the proto-oncogene RON. (A) High-throughput detection of splicing-effective mutations. Mutagenic PCR creates mutated minigene library (left) that gives rise to alternatively spliced transcripts (middle). Mutations and corresponding splicing products are characterised by DNA and RNA sequencing, respectively, and linked by unique 15-nt barcode sequence in each minigene (coloured boxes). Black and grey boxes depict constitutive and alternative exons, respectively. (B) Model-inferred landscapes of 1,747 single mutation effects on three different RON transcript isoforms with alternative exon (AE) inclusion, AE skipping and full intron retention (IR) in human HEK293T cells. Each mutation effect is indicated as a coloured dot according to the inserted nucleobase (see legend). Red lines indicate median (dashed) ± 2 standard deviations (SD; solid) for unmutated wild-type (wt) minigenes. [Images taken from Braun et al., 2018]](https://github.com/ZarnackGroup/ZarnackGroup.github.io/blob/main/Figure2_01.png)
